meta/downloadHelperBinaries.ts

```diff
@@ -59,6 +65,23 @@ async function downloadAsset(
   }
 }
 
+//FIXME: better name for this function
+async function downloadAsset2(
```

Comment:
What do I call this function, I don't want to use the same name as downloadAsset although that's what it does

Reply:
Why not just use the existing function & change the path used in Heroic?

Reply:
good idea